keda/README.md

```diff
@@ -152,3 +152,35 @@ be provided while installing the chart. For example,
 ```console
 helm install keda kedacore/keda --namespace keda -f values.yaml
 ```
+
+## KEDA is secure by default
+
+Our default configuration strives to be as secure as possible. Because of that, KEDA will run as non-root and be secure-by-default:
+```yaml
+podSecurityContext:
+  operator:
+    capabilities:
+      drop:
+      - ALL
+    allowPrivilegeEscalation: false
+    readOnlyRootFilesystem: true
+  metricServer:
+    capabilities:
+      drop:
+      - ALL
+    allowPrivilegeEscalation: false
+    ## Metrics server needs to write the self-signed cert so it's not possible set this
+    # readOnlyRootFilesystem: true
```

Comment:
```suggestion
    readOnlyRootFilesystem: true
  volumes:
    metricsApiServer:
      extraVolumes:
      - name: keda-volume
      extraVolumeMounts:
      - name: keda-volume
        mountPath: /apiserver.local.config/certificates/
```


@JorTurFer see updated instructions for readOnlyRootFilesystem=true in [FAQ](https://keda.sh/docs/2.8/faq/)

Should this secure snippet be added to the secure-by-default values?